### PR TITLE
Remove delay of loading sidebar items and other services on startup

### PR DESCRIPTION
**Pull request by winston-de**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Resolves issue discussed in discord server where the sidebar items are only loaded after the main page has loaded, which causes an unnecessary delay. 

**Details of Changes**
Add details of changes here.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

Tested in release mode and didn't see much of a difference. 

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: Files/App.xaml.cs [hunk 1]
```diff
@@ -196,12 +196,13 @@ protected override async void OnLaunched(LaunchActivatedEventArgs e)
...
             await logWriter.InitializeAsync("debug.log");
             Logger.Info($"App launched. Prelaunch: {e.PrelaunchActivated}");
-
+            
             //start tracking app usage
             SystemInformation.Instance.TrackAppUse(e);
 
             bool canEnablePrelaunch = ApiInformation.IsMethodPresent("Windows.ApplicationModel.Core.CoreApplication", "EnablePrelaunch");
 
+            LoadOtherStuffAsync().ContinueWith(t => Logger.Warn(t.Exception, "Error during LoadOtherStuffAsync()"), TaskContinuationOptions.OnlyOnFaulted);
```
Comment (d2dyno1) [inline Files/App.xaml.cs:205]: Should this be below `await EnsureSettingsAndConfigurationAreBootstrapped();` and in other places too?

**Review by d2dyno1 — CHANGES_REQUESTED** [review]

File: Files/App.xaml.cs [hunk 2]
```diff
@@ -155,13 +155,13 @@ private static async Task StartAppCenter()
             }
         }
 
-        public static async Task LoadOtherStuffAsync()
+        private static async Task LoadOtherStuffAsync()
```
Comment (duke7553) [inline Files/App.xaml.cs:158]:
```suggestion
        private static async Task InitializeAppAsync()
```
Do we want to use this opportunity to rename the method entirely?

Comment (winston-de) [reply Files/App.xaml.cs:158]: Yeah great idea. When I made this method a while back it was meant to be a temporary name.

File: Files/App.xaml.cs (hunk 1 above)
Comment (winston-de) [reply Files/App.xaml.cs:205]: Good idea

File: Files/App.xaml.cs (hunk 2 above)
Comment (d2dyno1) [reply Files/App.xaml.cs:158]: `InitializeAppComponentsAsync()`

**Review by d2dyno1 — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@winston-de thank you!

**Pull request merged**